lib/msal-core/src/utils/RequestUtils.ts

```diff
@@ -75,13 +78,40 @@ export class RequestUtils {
         return validatedRequest;
     }
 
+    /**
+     * 
+     * @param request 
+     * @param clientId 
+     * @param interactionType 
+     */
+    static validateLoginRequest(request: AuthenticationParameters, clientId: string, interactionType: InteractionType): AuthenticationParameters {
+        // Check if userRequest is empty
+        if (!request) {
+            request = {};
+        }
+
+        // Check if request scopes are null
+        if (!request.scopes) {
+            request.scopes = [];
+        }
+
+        // Append clientId to scopes by default for login calls
+        request.scopes = ScopeSet.appendScopes(request.scopes, [clientId]);
+
+        // validate request
+        const userRequest: AuthenticationParameters = RequestUtils.validateRequest(request, clientId, interactionType);
```

Comment:
Same comment as above about scopes:

```js
const requestScopes = (request && request.scopes) || [];
const scopes = ScopeSet.appendScopes(requestScopes, [ clientId ]);

const userRequest: AuthenticationParameters = RequestUtils.validateRequest({ ...request, scopes }, clientId, interactionType);

Reply:
Key in both these examples, I don't like conflation the falseyness of `request` with the need to mutate the passed in scopes. I think it is simpler to construct the needed scopes array outside the context of the request, and then handle the truthiness of that seperately.

Reply:
And the `...` operator takes care of that for us.

Reply:
Done.